src/less/skeleton-loader/skeleton-loader.less

```diff
@@ -0,0 +1,224 @@
+@import "../variables/variables.less";
+
+@skeleton-text-border-radius: 3px;
+@skeleton-text-height-small: 16px;
+@skeleton-text-height-large: 24px;
+
+:root {
+    --skeleton-default-height: 40px;
+    --skeleton-tile-width-min: 121px;
+    --skeleton-tile-width-max: 224px;
+}
+
+.skeleton {
+    background: var(--color-loading-fill);
+    cursor: progress;
```

Comment:
Is this progress cursor recommended by design or accessibility? I don't usually see this over skeleton loaders in other sites so it was a little surprising for me.

Reply:
When the mouse pointer communicates which action is happening beside not just displaying where it is, it adds to the experience. Often times when we interact with desktop applications, we notice these and I feel its lacking in web apps. As long as we are not degrading a good UX or a11y, I personally feel this is a good place to use the `cursor: progress` to show the state for sighted and mouse users.